src/UI/class-recommended-widget.php

```diff
@@ -109,27 +139,26 @@ public function widget( $args, $instance ): void {
 		echo wp_kses_post( $title_html );
 
 		// Set up the variables.
-		$options = $this->parsely->get_options();
 		$api_url = $this->get_api_url(
-			$options['apikey'],
+			$this->parsely->get_api_key(),
 			$instance['published_within'],
 			$instance['sort'],
 			$instance['boost'],
-			(int) $instance['return_limit']
+			$instance['return_limit']
 		);
 
 		$recommended_widget_script_asset = require plugin_dir_path( PARSELY_FILE ) . 'build/recommended-widget.asset.php';
 
 		?>
 
 		<div class="parsely-recommended-widget"
-			data-parsely-widget-display-author="<?php echo esc_attr( wp_json_encode( isset( $instance['display_author'] ) && $instance['display_author'] ) ); ?>"
```

Comment:
No need of null checks here because these will either have our defined values or default ones.